modules/n1ql/pages/n1ql-language-reference/with-recursive.adoc

```diff
@@ -212,8 +212,140 @@ SELECT * FROM rcte;
 ----
 ====
 
+.Examples handling hierarchical or tree-structured data
+====
+Before trying out these examples, you must:
```

Comment:
This won't work until you've created the `employees` collection. I'm not 100% of the best place to create this. It doesn't seem to belong in the `inventory` scope, so let's put it in `tenant_agent_00`.

Best practice in the SQL++ reference is to set the query context to the bucket and scope you'll be using for the query.

```suggestion
For this example, set the query context to the `tenant_agent_00` scope in the travel sample dataset.
For more information, see xref:n1ql:n1ql-intro/queriesandresults.adoc#query-context[Query Context].
```